automated install/basic-install.sh

```diff
@@ -1412,14 +1412,23 @@ installConfigs() {
         fi
         # and copy in the config file Pi-hole needs
         install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/${LIGHTTPD_CFG} "${lighttpdConfig}"
-        # Make sure the external.conf file exists, as lighttpd v1.4.50 crashes without it
-        if [ ! -f /etc/lighttpd/external.conf ]; then
-            install -m 644 /dev/null /etc/lighttpd/external.conf
-        fi
         # If there is a custom block page in the html/pihole directory, replace 404 handler in lighttpd config
         if [[ -f "${PI_HOLE_404_DIR}/custom.php" ]]; then
             sed -i 's/^\(server\.error-handler-404\s*=\s*\).*$/\1"\/pihole\/custom\.php"/' "${lighttpdConfig}"
         fi
+        # Copy the config file to include for pihole admin interface
+        if [[ -d "/etc/lighttpd/conf.d" ]]; then
+            install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/pihole-admin.conf /etc/lighttpd/conf.d/pihole-admin.conf
+        elif [[ -d "/etc/lighttpd/conf-available2" ]]; then
+            install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/pihole-admin.conf /etc/lighttpd/conf-available2/08-pihole-admin.conf
```

Comment:
```suggestion
            install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/pihole-admin.conf /etc/lighttpd/conf-available/08-pihole-admin.conf
```

Reply:
This appears to be a typo - I can't see any mention on the internet of `conf-available2`, either :)

Reply:
You're correct.  I was looking at the Debian package for lighttpd where some of the configs in the Debian repo are in conf-available2, e.g. https://salsa.debian.org/debian/lighttpd/-/tree/master/debian/conf-available2

I have fixed this and re-pushed.